ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/platform.j2

```diff
@@ -1,13 +1,41 @@
-{% if platform is defined and platform is not none %}
+{% if platform is arista.avd.defined %}
 
 # Platform
+{%     if platform.trident is arista.avd.defined or platform.sand is arista.avd.defined %}
 
 ## Platform Summary
-{%     if platform.trident is defined and platform.trident is not none %}
-{%       if platform.trident.forwarding_table_partition is defined and platform.trident.forwarding_table_partition is not none %}
+{%         if platform.trident is arista.avd.defined %}
 
-   The platform trident forwarding-table partition is {{ platform.trident.forwarding_table_partition }}.
-{%       endif %}
+### Platform Trident Summary
+
+| Settings | Value |
+| -------- | ----- |
+{%             for tridentsetting in platform.trident %}
+| {{ tridentsetting }} | {{ platform.trident[tridentsetting] }} |
+{%             endfor %}
+{%         endif %}
+{%         if platform.sand is arista.avd.defined %}
+
+### Platform Sand Summary
+
+| Settings | Value |
+| -------- | ----- |
+{%             for sandsetting in platform.sand %}
+{%                 if sandsetting == 'lag' %}
+{%                     for lag_sandsetting in platform.sand.lag %}
+{%                         set lag_sandsettingmod = "lag_" ~ lag_sandsetting %}
```

Comment:
I think it will make it more clear to use `.` as the seperator here. Same in line 31